aroquev/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -48,6 +38,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(json);
   }
 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // probaby verify that message is not empty
+    String comment = request.getParameter("comments-body-input");
+    comments.add(comment);
+    response.sendRedirect("/index.html");
```

Comment:
I think you can just redirect to "/" ? It might not make any difference.

Reply:
True, makes no difference. Done.